cookbooks/ambari/recipes/ambari_server_setup.rb

```diff
@@ -0,0 +1,37 @@
+#
+# Cookbook Name:: ambari
+# Recipe:: ambari_server_setup
+#
+# Copyright (c) 2016 Artem Ervits, All Rights Reserved.
+
+
+# /etc/ambari-server/conf/ambari.properties
+template "#{node['ambari']['ambari_server_conf_dir']}ambari.properties" do
+  source 'ambari.properties.erb'
+  owner 'root'
+  group 'root'
+  mode '0755'
+end
+
+# /etc/ambari-server/conf/password.dat
+template "#{node['ambari']['ambari_server_conf_dir']}password.dat" do
+  source 'password.dat.erb'
```

Comment:
This looks like a password, please use [`sensitive`](http://blog.thefourthparty.com/chef-sensitive/)